vtex/loaders/user.ts

```diff
@@ -38,6 +39,7 @@ async function loader(
       email: user.email,
       givenName: user.firstName,
       familyName: user.lastName,
+      taxID: user?.taxID?.replace(/[^\d]/g, ""),
```

Comment:
you might want `document` instead?
```suggestion
      taxID: user?.document?.replace(/[^\d]/g, ""),
```

Reply:
done!! thxx